src/Event/EventManager.php

```diff
@@ -213,9 +184,10 @@ public function off($eventKey, $callable = null)
             return $this;
         }
 
+        $callable = $callable(...);
         foreach ($this->_listeners[$eventKey] as $priority => $callables) {
             foreach ($callables as $k => $callback) {
-                if ($callback['callable'] === $callable) {
+                if ($callback['callable'] == $callable) {
```

Comment:
The `==` operator compares closures instead of having to match the exact format used with on().